engine/buildkit/executor_spec.go

```diff
@@ -427,8 +429,19 @@ func (w *Worker) setupRootfs(ctx context.Context, state *execState) error {
 	if err := mount.All(rootMnts, state.rootfsPath); err != nil {
 		return fmt.Errorf("mount rootfs: %w", err)
 	}
+
+	overlayIncompatDirs := overlay.VolatileIncompatDirs(rootMnts)
```

Comment:
For the curious (as I was) what an 'incompat' dir is: https://docs.kernel.org/filesystems/overlayfs.html#volatile-mount

Seems like a way to determine whether a mount should be discarded in the event of a rough shutdown. I guess it shouldn't matter much in our case, but that the right thing to do is to clean it up after use in the 'happy path'? The docs also say the "user" should notice its presence and discard the dirs, so I guess the OS doesn't do that itself?